modules/Sources/Dependencies/AddressBookClient/AddressBookLiveKey.swift

```diff
@@ -114,12 +122,12 @@ extension AddressBookClient: DependencyKey {
                 
                 syncedContacts.contacts.append($0)
                 
-                try storeContacts(syncedContacts, remoteStorage: remoteStorage)
+                let storeResult = try storeContacts(syncedContacts, remoteStorage: remoteStorage)
```

Comment:
```suggestion
                let storeResult = try storeContacts(syncedContacts, remoteStorage: remoteStorage, remoteStore: true)
```
(See my suggestion below for this not to have a default.)